### PR TITLE
[SMALLFIX] Moved the resource file of JsonJournalFormatterTest into package tachyon.master.journal

### PR DESCRIPTION
**Pull request by yupeng9**

### CODE REVIEW

File: servers/src/test/java/tachyon/master/journal/JsonJournalFormatterTest.java [hunk 1]
```diff
@@ -19,28 +19,29 @@
...
 
   @Override
   protected JournalFormatter getFormatter() {
     return new JsonJournalFormatter();
   }
 
-  @BeforeClass
-  public static void beforeClass() throws IOException {
+  @Before
+  public void beforeClass() throws IOException {
```
Comment (gpang) [inline servers/src/test/java/tachyon/master/journal/JsonJournalFormatterTest.java]: call this before()?

Comment (yupeng9) [inline servers/src/test/java/tachyon/master/journal/JsonJournalFormatterTest.java]: done

**Comment by yupeng9** [comment]
Jenkins, test this please

**Comment by uronce-cc** [comment]
+1 LGTM

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
LGTM

**Pull request merged**